src/components/ast/index.tsx

```diff
@@ -6,6 +6,7 @@ import { JsonAst } from "./json-ast";
 import { CssAst } from "./css-ast";
 import { MarkdownAst } from "./markdown-ast";
 import type { FC } from "react";
+import { HtmlAst } from "./html-ast";
```

Comment:
```suggestion
import { CssAst } from "./css-ast";
import { MarkdownAst } from "./markdown-ast";
import { HtmlAst } from "./html-ast";
import type { FC } from "react";
```

This is a minor stylistic suggestion! grouping the `***Ast` modules together would improve clarity.